backend/danswer/server/utils.py

```diff
@@ -1,9 +1,28 @@
 import json
+from datetime import datetime
 from typing import Any
 
 
-def get_json_line(json_dict: dict) -> str:
-    return json.dumps(json_dict) + "\n"
+class DateTimeEncoder(json.JSONEncoder):
+    """Custom JSON encoder that converts datetime objects to ISO format strings."""
+
+    def default(self, obj: Any) -> Any:
+        if isinstance(obj, datetime):
+            return obj.isoformat()
+        return super().default(obj)
+
+
+def get_json_line(json_dict: dict[str, Any]) -> str:
```

Comment:
Super nit:  I'd prefer passing DateTimeEncoder as the default argument to an encoder prop